source/LocationFilter.cpp

```diff
@@ -504,6 +505,150 @@ const Planet *LocationFilter::PickPlanet(const System *origin, bool hasClearance
 
 
 
+// Members of the NearOrDistanceFilter class.
+
+void LocationFilter::NearOrDistanceFilter::Load(const DataNode &node, int valueIndex)
+{
+	if(node.Size() > valueIndex)
+		LoadElement(node, valueIndex);
+	else if(node.HasChildren())
+		for(const auto &child : node)
+			LoadElement(child, 0);
+}
+
+
+
+LocationFilter::NearOrDistanceFilter::NearElement::NearElement(
+			const LocationFilter::NearOrDistanceFilter::DistanceElement &other)
+	: center(nullptr), minDistance(other.minDistance), maxDistance(other.maxDistance),
```

Comment:
Isn't center already initialized with nullptr in the class definition? (but then why do we need it if it's always null?)
```suggestion
	: minDistance(other.minDistance), maxDistance(other.maxDistance),
```